core/common/src/main/java/alluxio/retry/ExponentialBackoffRetry.java

```diff
@@ -52,7 +51,9 @@ protected long getSleepTime() {
       // current logic overflows at 30, so set value to max
       return mMaxSleepMs;
     } else {
-      int sleepMs = mBaseSleepTimeMs * Math.max(1, mRandom.nextInt(1 << (count + 1)));
+      // use randomness to avoid contention between many operations using the same retry policy
+      int sleepMs =
+          mBaseSleepTimeMs * (ThreadLocalRandom.current().nextInt(1 << count, 1 << (count + 1)));
```

Comment:
If there is no significant overhead and it has a better API I'm fine with using this instead of `Random` despite this not being accessed concurrently.